### PR TITLE
Prepare 0.23

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ and the [hyper HTTP library](https://github.com/hyperium/hyper).
 [![Documentation](https://docs.rs/hyper-rustls/badge.svg)](https://docs.rs/hyper-rustls/)
 
 # Release history
+- 0.23.0 (2021-15-11):
```
Comment (paolobarbolini) [inline README.md:11]:
```suggestion
- 0.23.0 (2021-11-15):
```

Comment (djc) [reply README.md]: Nice catch, thanks for the review!

**Comment by g2p** [comment]
There's a fix for a last minute mistake in #160 which should block this; my apologies.

**Comment by paolobarbolini** [comment]
Seeing https://github.com/rustls/hyper-rustls/pull/156#issuecomment-970127792 we should probably document the new feature flags

**Comment by djc** [comment]
@paolobarbolini sounds good, would you like to submit a PR against this branch?

**Comment by paolobarbolini** [comment]
Done in #161

**Review by paolobarbolini — APPROVED** [review]
FWIW I think this is ready to be released. Also I'd love to be able to remove another git dependency :smiley:

**Comment by djc** [comment]
Hopefully ctz will get to it this weekend.

**Pull request merged**